test/server/storage/safestorage.spec.js

```diff
@@ -456,6 +458,22 @@ describe('SafeStorage', function () {
                 .catch(done);
         });
 
+        it('should getCommits and get the one specified', function (done) {
```

Comment:
It is intended to be extra, it should not only get the commits, but also get the one specified.
But yes I agree, the phrasing is somewhat awkward. However since it was a getCommits test and the other such tests were named that way, I kept the 'should getCommits'.